readme.md

```diff
@@ -73,3 +73,55 @@ bot.once('spawn', () => {
 * Dynamic harvest/mining paths
 * Sprint
 * Actual move speed per block
+
+## API
+Considering there are a lot of deep changes that are being worked on, it could take some time before it's done
+
+Also, **for now**, there is only the `pathfinder` module, `movements` and `goals` still need to be done
+
+
+# Functions:
+
+### bot.pathfinder.bestHarvestTool(block)
+Returns the best harvest tool in the inventory for the specified block
+ * `Return` - ?
+ * `block` - Block instance
```

Comment:
might also be a good idea to hyperlink some documentation on this `Block` instance